bin/esy.re

```diff
@@ -233,6 +233,55 @@ let buildDependencies = (all, mode, pkgarg, proj: Project.t) => {
   Project.withPackage(proj, pkgarg, f);
 };
 
+let gc = (projCfgs: list(ProjectConfig.t), dryRun) => {
+  open RunAsync.Syntax;
+  let mode = BuildSpec.BuildDev;
+  let%bind (installedBuilds, allCachedBuilds) =
+    RunAsync.List.foldLeft(
+      ~init=(PathSet.empty, PathSet.empty),
+      ~f=
+        (acc, projCfg) => {
+          let (installedBuilds, allCachedBuilds) = acc;
```

Comment:
I'd call those `dirsToKeep` and `allDirs` — as I understand this is not just builds, but also staging dirs and others.

Reply:
Done!